protocols/bgp/packet/path_attributes_test.go

```diff
@@ -2383,3 +2383,37 @@ func TestDecode4ByteASN(t *testing.T) {
 		assert.Equal(t, test.expected, res, test.name)
 	}
 }
+
+func TestPathAttributeCopy(t *testing.T) {
+	attr := &PathAttribute{
```

Comment:
Can you make up a PathAttribute with every Field not set to a default value? This ensures every field is copied and not set to the default value.

Reply:
I've added every field that exists for now. However, the test will not fail if someone adds a new field to the struct and forgets to copy its value in the `Copy()` function. Do you think I should test that scenario using reflection or does that sound like a bad, over-engineered idea?